man/systemd-repart.xml

```diff
@@ -440,6 +440,14 @@
         due to missing permissions.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--copy-from=</option><arg>IMAGE</arg></term>
+
+        <listitem><para>Instructs <command>systemd-repart</command> to copy the partitions from the given
+        image. The partitions from the given image are synthesized into partition definitions that are
+        parsed before the partition definition files.</para></listitem>
```

Comment:
i cannot parse this.

"partitions … are synthesized into … definitions that are parsed"?

what's that supposed to mean?

Reply:
so let me grok this: this basically copies *all* partitions from the specified source 1:1, i.e. the contents and metadata will be identical, but the offset and partno will likely be different? please say so explicitly. I had a hard time figuring out what this does.